test/kitchen_sink/kitchen_sink.c

```diff
@@ -41,6 +41,7 @@
 #include "pico/malloc.h"
 #include "pico/multicore.h"
 #include "pico/printf.h"
+#include "pico/rand.h"
```

Comment:
`#include "pico/rand.h"` now appears both here and also on line 51

Reply:
crap; ok; tsk people not keeping thing is alphabetical order :-(